mob.go

```diff
@@ -122,6 +122,19 @@ func (branch Branch) hasRemoteBranch(configuration config.Configuration) bool {
 	return false
 }
 
+func (branch Branch) hasLocalBranch(localBranches []string) bool {
```

Comment:
Imo, hasLocalBranch should not need an argument localBranches. It can fetch them itself. And if local branches are fetched multiple times, I still think its worth it. When I need to find out how to check if a localBranch exists in our code, and I find this function, I don't want it to make me search for yet another function.

Reply:
The same is probably applicable to determineBranches